Example/Controllers/ContactsListSettingsTableViewController.swift

```diff
@@ -1,46 +1,96 @@
 import UIKit
 import MiniApp
 
+protocol ContactsListDelegate: class {
+    func contactsController(_ contactsController: ContactsListSettingsTableViewController, didSelect contact: [MAContact]?)
+}
+
 class ContactsListSettingsTableViewController: UITableViewController {
 
     var userContactList: [MAContact]? = []
-    internal static let contactName = "Example Name", contactEmail = "name@example.com"
+    weak var contactDelegate: ContactsListDelegate?
+    var selectedContacts = [MAContact]()
+    var allowMultipleSelection = true
 
     override func viewDidLoad() {
         super.viewDidLoad()
         prepareRandomContactList()
         self.tableView.separatorStyle = .singleLine
-        self.tableView.tableFooterView = UIView(frame: .zero)
```

Comment:
the frame is added in storyboard. I needed a bigger one to add a button on the bottom of the screen